common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -95,6 +103,20 @@ public static Map<String, Object> fromJson(String jsonStr, String defaultKey) {
         return result;
     }
 
+    public static Map<String, String> filterInterfaceMap(Map<String, ?> interfaceObjs) {
```

Comment:
Why put this model interface specific method in this generic StringUtils class ?

Or why build this function specially for model interface ? If you want to put it in this generic StringUtils class, please make this function generic too. Otherwise you can move to MLModel class

Reply:
Re-write to a more generic method

Reply:
Just use `map.get("input/output")` and you can save object creation and extra for loops.